### PR TITLE
feat: 🎸 Hero homepage

### PR DESCRIPTION
**Pull request by MatthysDev**
✅ Closes: #157

### CODE REVIEW

File: src/components/hero.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        <div className="flex-1 mb-4 md:mb-0 h-52 md:h-full">
+          <Link to={`/podcast/${allEpisodes[0].guid}`}>
+            <div
+              className={imgStyle}
+              style={{
+                backgroundImage: `url(${allEpisodes[0].itunes.image})`,
+              }}
+            >
+              <Text as="h3" className={textStyle}>
+                {allEpisodes[1].title}
```
Comment (flexbox) [inline src/components/hero.tsx:28]: should be

```
{allEpisodes[0].title}
```

File: src/components/hero.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+              </Text>
+            </div>
+          </Link>
+        </div>
+        <div className="flex-1 flex flex-col gap-4">
+          <div className="bg-cover bg-center h-52 md:h-1/2 ">
+            <Link to={`/podcast/${allEpisodes[1].guid}`}>
```
Comment (flexbox) [inline src/components/hero.tsx:35]: Why haven't you created a component like

```
<HeroCard
  slug={}
  title={}
  backgroundImage={}
  size="xl" <---- could be sm or md or xl
/>
```

Use cva for the variant

Comment (MatthysDev) [reply src/components/hero.tsx:35]: cva broke everythings
CleanShot 2023-09-14 at 00 51 05@2x

File: src/components/heroCard.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+export const HeroCard: React.FC<HeroCardProps> = ({
+  className,
+  imageUrl,
+  size,
+  heroLink,
+  heroTitle,
+  ...rest
+}) => {
+  const textStyle =
```
Comment (flexbox) [inline src/components/heroCard.tsx:37]: I don't think we need these 2 variables anymore, we can do
```
className="text-white bg-black items-end flex  mb-6 mx-auto p-2 w-5/6 text-left"
```

**Review by flexbox — CHANGES_REQUESTED** [review]
- [x] Drop this section

image

- [x] Add a h1 title for SEO "Podcasts récents sur les artistes femmes"
image

**Review by flexbox — APPROVED** [review]
Sounds Good To Me 👍

**Review by flexbox — APPROVED** [review]

**Pull request merged**